src/services/createVisa.ts

```diff
@@ -0,0 +1,10 @@
+import _ from 'lodash';
+import ajax from 'services/ajax';
+
+const BLOCKED_KEYS = ['id', 'createdAt', 'lastLogin', 'groups', 'applications'];
+
+export const createVisa = ({ item }) => {
+  return ajax.post(`/visa`, _.omit(item, BLOCKED_KEYS)).then(r => {
```

Comment:
It is probably better to `pick` just the keys you want instead of omitting a long list. Better to be specific about what we include in most cases.

Not an issue that needs changing, just a thought.